bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3977,6 +4083,7 @@ ReplicaImp::ReplicaImp(const ReplicaConfig &config,
   auto numThreads = 8;
   LOG_INFO(GL, "Starting internal replica thread pool. " << KVLOG(numThreads));
   internalThreadPool.start(numThreads);  // TODO(GG): use configuration
+  postExecThread_.start(1);              // TODO(LG) should be configurable;
```

Comment:
Should not be in two places, if possible. Could it be moved to the common constructor function called by both other constructors?

Reply:
Done, thanks.